### PR TITLE
NXLog LinuxAudit data connector: Initial Commit

### PR DESCRIPTION
**Pull request by techwriter-dev**
# Initial submission of the NXLog LinuxAudit data connector

  1. Connector UX:  `NXLogLinuxAudit.json`
  2. Sample Data:    `NXLogLinuxAudit_CL.json`
  3. Logo:                 ` NXLog.svg`

### CODE REVIEW

File: DataConnectors/NXLogLinuxAudit.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Most frequent type",
+            "query": "LinuxAudit_CL\n| summarize EventCount = count() by type_s \n| where strlen(type_s) > 1 \n| render barchart"
+        },
+        {
+            "description" : "Most frequent comm",
+            "query": "LinuxAudit_CL\n| summarize EventCount = count() by comm_s\n| where strlen(comm_s) > 1\n| render barchart"
```
Comment (v-jayakal) [inline DataConnectors/NXLogLinuxAudit.json:20]: Could not find field with name comm_s. Can you please fix this query?

File: DataConnectors/NXLogLinuxAudit.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
+{
+    "id": "NXLogLinuxAudit",
+    "title": "NXLog LinuxAudit",
+    "publisher": "NXLog",
+    "descriptionMarkdown": "The NXLog [LinuxAudit](https://nxlog.co/documentation/nxlog-user-guide/im_linuxaudit.html) data connector supports custom audit rules and collects logs without auditd or any other user-space software. IP addresses and group/user ids are resolved to their respective names making [Linux audit](https://nxlog.co/documentation/nxlog-user-guide/linux-audit.html) logs more intelligible to security analysts. This REST API connector can efficiently export Linux security events to Azure Sentinel in real-time.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "LinuxAudit_CL",
+            "baseQuery": "LinuxAudit_CL"
```
Comment (v-jayakal) [inline DataConnectors/NXLogLinuxAudit.json:10]: As per the json provided in sample data, custom table name is NXLogLinuxAudit_CL. Can you please either change the file name here https://github.com/Azure/Azure-Sentinel/pull/1280/files#diff-ac84e6e8f76926156578d14386bf7e10086ea3ba9901c2c60bbf00b86307f3a8 to LinuxAudit_CL or please use NXLogLinuxAudit_CL in all queries (sample queries, graph queries, datatypes and connectivitycriteria) ?

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/NXLogLinuxAudit.json (hunk 2 above)
Comment (techwriter-dev) [reply DataConnectors/NXLogLinuxAudit.json:10]: This was resolved in the latest commit [08dec290](https://github.com/Azure/Azure-Sentinel/commit/08dec290af28748496d5ed66779cf2fef6718dab).

File: DataConnectors/NXLogLinuxAudit.json (hunk 1 above)
Comment (techwriter-dev) [reply DataConnectors/NXLogLinuxAudit.json:20]: @v-jayakal This was resolved in the latest commit [08dec290](https://github.com/Azure/Azure-Sentinel/commit/08dec290af28748496d5ed66779cf2fef6718dab).

File: DataConnectors/NXLogLinuxAudit.json (hunk 2 above)
Comment (v-jayakal) [reply DataConnectors/NXLogLinuxAudit.json:10]: Could not find the updates of the commit 08dec290 in this branch. Can you please merge the changes to this branch? 
NxLogCommit

**Comment by techwriter-dev** [comment]
@v-jayakal  The [changes are now in the `master` branch](https://github.com/techwriter-dev/Azure-Sentinel/blob/master/Sample%20Data/Custom/LinuxAudit_CL.json) of my forked repository.

Thanks!

**Comment by techwriter-dev** [comment]
@v-jayakal Help! I think I closed this Pull Request by mistake. I am not used to using GitHub anymore since we typically use GitLab. I though I was closing/resolving the thread for your last Pull Request change.

**Review by v-jayakal — CHANGES_REQUESTED** [review]
As you mentioned in NXLog DNS Logs, pull request https://github.com/Azure/Azure-Sentinel/pull/1298/files will be ignored as this PR has the files for NXLog DNS Logs connector.

File: DataConnectors/NXLogDnsLogs.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
+{
+    "id": "NXLogDnsLogs",
+    "title": "NXLog DNS Logs",
+    "publisher": "NXLog",
+    "descriptionMarkdown": "The NXLog DNS Logs data connector uses Event Tracing for Windows ([ETW](https://docs.microsoft.com/windows/apps/trace-processing/overview)) for collecting both Audit and Analytical DNS Server events. The [NXLog *im_etw* module](https://nxlog.co/documentation/nxlog-user-guide/im_etw.html) reads event tracing data directly for maximum efficiency, without the need to capture the event trace into an .etl file. This REST API connector can forward DNS Server events to Azure Sentinel in real-time.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "DNS_Logs_CL",
```
[the remark this answers is not in the document]
Comment (v-jayakal) [reply DataConnectors/NXLogDnsLogs.json:9]: Resolving as the file name has changed to DNS_Logs_CL.

**Review by v-jayakal — APPROVED** [review]
Closing https://github.com/Azure/Azure-Sentinel/pull/1298 - NXLog DNS Logs data connector to avoid duplicates as NXLog DNS Logs connector files are also here.

**Review by nazang — APPROVED** [review]

**Pull request merged**